lib/iris/_merge.py

```diff
@@ -36,8 +36,8 @@
 import iris.cube
 import iris.coords
 import iris.exceptions
-from iris._lazy_data import is_lazy_data, as_concrete_data, as_lazy_data
 import iris.util
+from iris._lazy_data import is_lazy_data, array_masked_to_nans
```

Comment:
@marqh Minor point but `from iris._lazy ...` sorts before alphabetical, and should be the first `iris` import.